gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
I was suggesting that:

> command || return

is fairly "common pattern" in bash and we can find many references to this pattern it in documents. 

My intention was not to go into specifics with exit code and or discussing with gnu vs bsd sort.

Any proper fix is fine with me.